packages/wonder-blocks-modal/components/modal-launcher.js

```diff
@@ -71,7 +71,7 @@ export default class ModalLauncher extends React.Component<Props, State> {
         this.setState({opened: true});
     };
 
-    _closeModal = () => {
+    handleCloseModal = () => {
```

Comment:
We could change the prefix to `_handle` if we want to, but then we'd have to use `_`s for all handlers which is a bigger change.

Reply:
Hmm if we have a style of underscoring private methods anyway, then making the prefix be `_handle` seems appropriate. But I'm not sure that's a style we use consistently, or whether we should…

Reply:
It's not really clear to me what constitutes a private method in a React class. You almost never call a method directly on a React class so should all methods that aren't the constructor, render, or lifecycle methods be marked as private?

Reply:
That's how I treat it, yeah :) You sometimes _do_ call methods directly on a React class, via refs, so I suppose that's the only place where the distinction really matters. (I also like it for further distinguishing between custom methods vs lifecycle methods, but I suppose that placement and other IDE tricks can do that just as effectively.)

So yeah, not sure what we want here!

Reply:
That's how I treat it, yeah :) You sometimes _do_ call methods directly on a React class, via refs, so I suppose that's the only place where the distinction really matters. I also like it for consistency with how I name things in other JS classes, but we don't have many of those tbh. And I also like it for further distinguishing between custom methods vs lifecycle methods, but I suppose that placement and other IDE tricks can do that just as effectively!

So yeah, not sure what we want here!

Reply:
Our React style guide makes no mention of private methods and the prefix it uses for handlers is `handle` so I think we should go with that for now until there's an ADR to change it.

Reply:
Yeah, I've never been sure why we were doing _ prefixing as it's really hard to dip into the methods of a React class in the first place!